src/operators/operators.jl

```diff
@@ -1,61 +1,39 @@
-### AbstractSciMLOperator Interface
-
-#=
-1. Function call and multiplication: L(du, u, p, t) for inplace and du = L(u, p, t) for
-   out-of-place, meaning L*u and mul!(du, L, u).
-2. If the operator is not a constant, update it with (u,p,t). A mutating form, i.e.
-   update_coefficients!(A,u,p,t) that changes the internal coefficients, and a
-   out-of-place form B = update_coefficients(A,u,p,t).
-3. isconstant(A) trait for whether the operator is constant or not.
-4. islinear(A) trait for whether the operator is linear or not.
-=#
-
-Base.eltype(::Type{AbstractSciMLOperator{T}}) where {T} = T
-Base.eltype(::AbstractSciMLOperator{T}) where {T} = T
-update_coefficients!(L, u, p, t) = nothing
-update_coefficients(L, u, p, t) = L
+#
+#Base.eltype(::Type{AbstractSciMLOperator{T}}) where {T} = T
+#Base.eltype(::AbstractSciMLOperator{T}) where {T} = T
+#update_coefficients!(L, u, p, t) = nothing
+#update_coefficients(L, u, p, t) = L
```

Comment:
No commented stuff like this, please clean.

Reply:
done